db/active_replicator_checkpointer.go

```diff
@@ -422,17 +430,22 @@ func (c *Checkpointer) fetchCheckpoints() (*ReplicationStatus, error) {
 		c.stats.GetCheckpointHitCount++
 	}
 
-	c.lastCheckpointSeq = checkpointSeq
+	parsedCheckpointSeq, err := ParsePlainSequenceID(checkpointSeq)
+	if err != nil {
```

Comment:
On a failed parse, do you think we should be setting the sequence to zero for rollback, like we do above?  Otherwise, is this situation recoverable?

Reply:
We're able to warn and use seq:0 so I've done that and consider it a checkpoint miss for stats